### PR TITLE
Improved installation docs

### PR DESCRIPTION
**Pull request by twentyse7en**
Generalised virtualenv creation for different
python version.
Added a optional instruction for installing pytorch.

## Contributor Checklist:

* [x] I have added [a changelog entry](https://github.com/LiberTEM/LiberTEM/tree/master/docs/source/changelog) for my contribution

### CODE REVIEW

File: docs/source/install.rst [hunk 1]
```diff
@@ -58,7 +58,11 @@ To create a new virtualenv for LiberTEM, you can use the following command:
 
 .. code-block:: shell
 
-    $ virtualenv -p python3.7 ~/libertem-venv/
+    $ virtualenv -p python3.x ~/libertem-venv/
+
+Replace :code:`x` with current Python version installed. To find out which
+version of Python is installed on your system run the :code:`python --version`
+or :code:`python -V` command.
```
Comment (sk1p) [inline docs/source/install.rst]: Hmm, this is kind of redundant. If you do `python --version` and put that into the `virtualenv` command, you could as well do `virtualenv -p python ~/libertem-venv/` (or even leave off the `-p` switch entirely).

Also note that the `python` binary can point to either Python 2 or Python 3 - this is detailed in [PEP 394](https://www.python.org/dev/peps/pep-0394/#for-python-runtime-distributors). So if I followed your instructions on my system, I would get a Python 2 virtualenv, which is not compatible with LiberTEM. To make sure you get Python 3, you can use the `python3` binary:

>We expect Unix-like software distributions (including systems like macOS and Cygwin) to install the python2 command into the default path whenever a version of the Python 2 interpreter is installed, and the same for python3 and the Python 3 interpreter

The point of using `-p python3.7` was to get a compatible Python version, not just any installed python3. So maybe we should use `-p python3` as the easiest way that works on most systems, but point out that you need to substitute a compatible version if you have multiple python versions installed. For example, I have Python 3.6, 3.7 and 3.8 installed on my system, so they would be all compatible, but if someone had Python 3.5 and 3.6 installed, they possibly would have to explicitly use `python3.6` to get LiberTEM running.

Note that the README.rst also contains virtualenv creation, so it should be changed there, too.

File: docs/source/changelog/documentation/venv_doc.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[Documentation] virtualenv instruction improved
+===============================================
+
+ * Instruction generalised for different python
+   version.
+ * optional pytorch installation
```
Comment (sk1p) [inline docs/source/changelog/documentation/venv_doc.rst]: pytorch installation was in docs before, describe a bit more what you changed

File: docs/source/install.rst [hunk 3]
```diff
@@ -198,6 +202,13 @@ example from PyPi:
 
     (libertem) $ pip install libertem[torch]
 
+Some shell interprets above code differently and might cause an error, in this
+scenario quoting the package name argument might help.
+
+.. code-block:: shell
+
+    (libertem) $ pip install "libertem[torch]"
+
```
[the remark this answers is not in the document]
Comment (twentyse7en) [reply docs/source/install.rst]: Thats right, also too many codes way raise confusion too!

File: docs/source/install.rst (hunk 1 above)
Comment (twentyse7en) [reply docs/source/install.rst]: Thanks! Such a wonderful explanation. Made some changes, hope that reflect what you mentioned.

File: README.rst [hunk 4]
```diff
@@ -47,13 +47,13 @@ The short version:
 
 .. code-block:: shell
 
-    $ virtualenv -p python3.6 ~/libertem-venv/
+    $ virtualenv -p python3 ~/libertem-venv/
     $ source ~/libertem-venv/bin/activate
     (libertem) $ pip install libertem[torch]
```
Comment (sk1p) [inline README.rst]: This line should also have quotes, like the corresponding line in the documentation.

Comment (twentyse7en) [reply README.rst]: Oops! forgot that one, Fixed :smile:

**Comment by sk1p** [comment]
Thanks, will merge once CI passes!

**Pull request merged**